hooks/react-client/useKeyboardCommands.ts

```diff
@@ -0,0 +1,38 @@
+import { useEffect } from 'react';
+
+type KeyboardCommand = 'cmd-k' | 'escape' | 'down' | 'up' | 'enter';
+
+type KeyboardCommandCallback = (key: KeyboardCommand) => void;
+
+const useKeyboardCommands = (fn: KeyboardCommandCallback) => {
+  useEffect(() => {
+    document.addEventListener('keydown', event => {
+      // Detect ⌘ + k on Mac, Ctrl + k on Windows
+      if ((event.metaKey || event.ctrlKey) && event.key === 'k') {
```

Comment:
Note that this means that this also listens to the Windows Start menu Key (because on Windows metaKey is the Windows button)

Reply:
What's the suggested key then?

Reply:
There's no easy fix. Just saying, and might be worth mentioning there that there's a sideEffect on both OSs.

Reply:
yeah... I fear I'll need some help from someone using Windows. I only have access to OSX and Linux

Reply:
I'll resolve this for now, we could track this in a separate issue